docs/pages/docs/custom-schemas/custom-blocks.mdx

```diff
@@ -77,76 +41,108 @@ const Alert = createReactBlockSpec(
 );
 ```
 
-You can see that `createReactBlockSpec` takes two object arguments:
+#### Block Config (`CustomBlockConfig`)
```

Comment:
Not sure about putting the type in brackets - consumers shouldn't have to care about it anyway right? They just need to know what they should put in `createReactBlockSpec`

Reply:
it links back to the earlier code snippet (type definition), where `CustomBlockConfig` is used)